### DISCUSSION TITLE
Does allantools have a changelog for versions?

### DISCUSSION THREAD

**Discussion by aidanmontare-fed** [aewallin/allantools#155] [Q&A]
/ is this something that would be of interest?

**Comment by aewallin** [comment]
There's a list of main changes at the start of allantools.py:
https://github.com/aewallin/allantools/blob/master/allantools/allantools.py#L10

That list could be moved into a changelog-file, and possibly included in the generated documentation on readthedocs? I don't have any more detailed notes of what changed in which version..

**Reply by aidanmontare-fed** [reply]
Thanks! Would you like me to start on this?

**Reply by aidanmontare-fed** [reply]
It looks like some of the other source files have the beginnings of changelogs in them. Alternatively to a global changelog, we could keep the changelogs in each file (and work on fleshing them out). Do we have any preference on this?

**Reply by aidanmontare-fed** [reply]
And if it's not all that pressing to have a changelog for older versions, we could just start one starting with the next release and going forward, which would save the work of going through all the old changes.
